### PR TITLE
Create develompent item option

### PR DESCRIPTION
**Pull request by MistrKernnunos**

### CODE REVIEW

File: ProjectOptions.cmake [hunk 1]
```diff
@@ -255,3 +259,5 @@ else()
   set(BOOTLOADER_UPDATE NO)
 endif()
 define_boolean_option(BOOTLOADER_UPDATE ${BOOTLOADER_UPDATE})
+
+define_boolean_option(DEVELOPMENT_ITEMS ${DEVELOPMENT_ITEMS_ENABLED})
```
Comment (dragomirecky) [inline ProjectOptions.cmake:263]: move up

File: ProjectOptions.cmake [hunk 2]
```diff
@@ -85,6 +85,10 @@ set(CONNECT
     "YES"
     CACHE BOOL "Enable Connect client"
     )
+set(DEVELOPMENT_ITEMS_ENABLED
+    NO
```
Comment (dragomirecky) [inline ProjectOptions.cmake:89]: default yes

File: utils/holly/build-pr.jenkins [hunk 3]
```diff
@@ -43,7 +43,7 @@ pipeline {
                 script {
                     // required configurations
                     def configurations = [
-                        [preset: "mini", build_type: "release", bootloader: "yes"],
+                        [preset: "mini", build_type: "release", bootloader: "yes",devel_items: "no"],
```
Comment (dragomirecky) [inline utils/holly/build-pr.jenkins:46]: build "yes" variant normally, "no" for release branch only

**Review by dragomirecky — COMMENTED** [review]
comments based on in-person review

File: ProjectOptions.cmake (hunk 1 above)
Comment (MistrKernnunos) [reply ProjectOptions.cmake:263]: 👌🏻

File: ProjectOptions.cmake (hunk 2 above)
Comment (MistrKernnunos) [reply ProjectOptions.cmake:89]: 👌🏻

File: utils/holly/build-pr.jenkins (hunk 3 above)
Comment (MistrKernnunos) [reply utils/holly/build-pr.jenkins:46]: 👌🏻

**Review by dragomirecky — APPROVED** [review]
looks good 👏

**Pull request merged**